colors/default.kak

```diff
@@ -6,6 +6,7 @@ face global type yellow
 face global variable green
 face global module green
 face global function cyan
+face global functionPrivate yellow
```

Comment:
I am not sure about adding this as part of the default faces, you could introduce an `ElixirPrivateFunction` directly inside elixir.kak that would default to some modification of the function, (say `default+i@function` to add italic to whatever function is in the current colorscheme).